packages/plugins/rubygem-smart-proxy-probing/rubygem-smart-proxy-probing.spec

```diff
@@ -1,44 +1,51 @@
-# Generated from smart-proxy-probing-0.0.2.gem by gem2rpm -*- rpm-spec -*-
 # template: smart_proxy_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%{!?_root_datadir:%global _root_datadir %{_datadir}}
+%{!?_root_localstatedir:%global _root_localstatedir %{_localstatedir}}
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
 %global gem_name smart-proxy-probing
 %global plugin_name smart-proxy-probing
 
-%global foreman_proxy_min_version 1.17
-%global foreman_proxy_dir %{_datarootdir}/foreman-proxy
+%global foreman_proxy_min_version 1.24
+%global foreman_proxy_dir %{_root_datadir}/foreman-proxy
```

Comment:
it kinda feels like these (here up to `smart_proxy_dynflow_bundlerd_dir`) globals should be defined in `foreman-proxy` instead of copying them in every spec file?

wouldn't block on this, tho

Reply:
I did debate making macros for these things. That would require adding stuff to the buildroot so I might err on the side of doing it in foreman RPM. I'm trying to resolve some of this through re-design of smart proxy dynflow core

Reply:
yeah, that works too.